zebra/if_netlink.c

```diff
@@ -77,6 +77,9 @@ set_ifindex(struct interface *ifp, ifindex_t ifi_index, struct zebra_ns *zns)
 		     "was renamed from %s to %s, results are uncertain!",
                     ifi_index, oifp->name, ifp->name);
 	  if_delete_update(oifp);
+	  if (IS_ZEBRA_DEBUG_KERNEL)
+	    zlog_debug ("delete pseudo iface %s(%u) from zebra", oifp->name, ifi_index);
```

Comment:
Nit -- but can we spell out "interface" in the various places "iface" is used? It seems to be spelled out in all the other output; this would make the output more consistent for folks using scripts to "screen scrape" stuff.